src/azure-cli/azure/cli/command_modules/vm/_help.py

```diff
@@ -3396,14 +3396,13 @@
 helps['vmss reimage'] = """
 type: command
 short-summary: Reimage VMs within a VMSS.
-parameters:
-  - name: --instance-id
-    short-summary: VM instance ID. If missing, reimage all instances.
 examples:
-  - name: Reimage VMs within a VMSS. (autogenerated)
+  - name: Reimage a VM instance within a VMSS.
     text: |
-        az vmss reimage --instance-id 1 --name MyScaleSet --resource-group MyResourceGroup --subscription MySubscription
-    crafted: true
+        az vmss reimage --instance-ids 1 --name MyScaleSet --resource-group MyResourceGroup --subscription MySubscription
+  - name: Reimage a batch of VM instances within a VMSS.
+    text: |
+        az vmss reimage --instance-ids 1 2 3 --name MyScaleSet --resource-group MyResourceGroup --subscription MySubscription
```

Comment:
Let's provide an example for reimaging all instances.

Reply:
OK, done